Prioritize/Java/Single Project/Gradle/prioritize.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#### Prerequisite commands & installs
+# apt-get update
+# apt-get install -y curl git openjdk-8-jdk nano
+
+#### Clone your repo & run script
+# git clone <your repository> && cd ./<your repository>
+# chmod +x ./prioritize.sh
+# ./prioritize.sh
+
+#### Build application & check JAVA_HOME
+echo  $JAVA_HOME
+./gradlew build -x test
+
+#### Run WS Prioritize
+curl -LJO https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
```

Comment:
Consider using the s3 URL, for consistency:
curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar